toolz/itertoolz.py

```diff
@@ -496,6 +496,34 @@ def concatv(*seqs):
     return concat(seqs)
 
 
+def duplicate(seq, n=1):
+    """ Gets each element multiple times.
+
+    Like ``itertools.repeat`` this will repeat each element n-times.
+    However, it will do this for each element of the sequence.
+
+     Args:
```

Comment:
Toolz currently does not have strictly formatted docstrings. The motivation for this is that the functions should be simple enough to show with some prose and a few examples: https://github.com/pytoolz/toolz/pull/251#r35478952